cluster-autoscaler/cloudprovider/mcm/ec2_instance_types.go

```diff
@@ -0,0 +1,2505 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file was generated by go generate; DO NOT EDIT
```

Comment:
you generated it using a script right. I assume its just a copy paste?

Reply:
I have copy pasted the license and used a python to script to modify the rest of the entries.

Reply:
yes, I just wanted to confirm as its written `generated by go generate` , Thanks